src/java/main/org/apache/zookeeper/server/FinalRequestProcessor.java

```diff
@@ -460,11 +464,21 @@ public void processRequest(Request request) {
             if (request.type == OpCode.closeSession) {
                 cnxn.sendCloseSession();
             }
+            checkLatency(request);
         } catch (IOException e) {
             LOG.error("FIXMSG",e);
         }
     }
 
+    private void checkLatency(Request request) {
+        long requestWarnThresholdMs = QuorumPeerConfig.getRequestWarnResponseThresholdMs();
```

Comment:
I am thinking to cache the value of `requestWarnThresholdMs` inside the `FinalRequestProcessor.java`. Is that a good idea?